src/ScenarioManagerTreeList/ScenarioManagerTreeList.js

```diff
@@ -93,22 +121,11 @@ const ScenarioManagerTreeList = (props) => {
 ScenarioManagerTreeList.propTypes = {
   datasets: PropTypes.array.isRequired,
   scenarios: PropTypes.array.isRequired,
-  moveScenario: PropTypes.func.isRequired
+  deleteScenario: PropTypes.func.isRequired,
+  moveScenario: PropTypes.func.isRequired,
+  userId: PropTypes.string
```

Comment:
UserId can be required?
As scenario deletion is allowed only for scenarios that you have created, the user info is needed.

Reply:
Done :+1: